fdbclient/NativeAPI.actor.cpp

```diff
@@ -169,6 +169,34 @@ void DatabaseContext::removeTssMapping(StorageServerInterface const& ssi) {
 	}
 }
 
+void DatabaseContext::addSSIdTagMapping(const UID& uid, const Tag& tag) {
+	ssidTagMapping[uid] = tag;
+}
+
+void DatabaseContext::getLatestCommitVersions(const Reference<LocationInfo>& locationInfo,
+                                              Version readVersion,
+                                              VersionVector& latestCommitVersions) {
+	std::map<Version, std::set<Tag>> versionMap; // order the versions to be returned
+	for (int i = 0; i < locationInfo->locations()->size(); i++) {
+		UID uid = locationInfo->locations()->getId(i);
+		if (ssidTagMapping.find(uid) != ssidTagMapping.end()) {
```

Comment:
is it expected to not be able to find the `uid` in `ssidTagMapping`

Reply:
> is it expected to not be able to find the `uid` in `ssidTagMapping` or would that be an error?

I don't know. If the proxy always has the most up-to-date UID-> Tag mapping then we should always find an entry here. Are there any exceptions to this on the proxy? Maybe Jingyu or Marcus can answer this question.